gtm/README.md

```diff
@@ -0,0 +1,141 @@
+# Search Insights for Google Tag Manager
+
+Google Tag Manager [custom template](https://developers.google.com/tag-manager/templates/) for Algolia Search Insights.
+
+---
+
+<div align="center">
+
+[**Download template**](generated/search-insights.tpl)
+
+</div>
+
+---
+
+## Usage
+
+### 1. Import the template
+
+- Download **[`search-insights.tpl`](generated/search-insights.tpl)**
+- Import the file in [Google Tag Manager](https://tagmanager.google.com) (GTM)
+  - Go to "Templates"
+  - Click on "New"
+  - Select "Import" in the menu
+
+### 2. Create variables
```

Comment:
Can we help out the user more here, by explaining which variables need to be added, and where to find them?

Reply:
Sure, we can guide them more. For now I didn't want to give too many indications and "good practices" because I'm myself not entirely sure what to recommend, hence the link to Simo Ahava's blog. Let's wait a bit what patterns emerge before recommending too many things (custom templates are still quite new).

Reply:
I just meant listing the variables they need to create and tell them were to find them, like you did for queryID for example. Does this need to wait for patterns to emerge?

Reply:
69def5e9a9aadfa5dd433a59a70ca20a42f4f712